### PR TITLE
move include("SDPutils.jl") inside the top-level package module

### PR DESCRIPTION
**Pull request by tkelman**
otherwise you're creating two top-level modules in the package instead of one

### CODE REVIEW

**Comment by trigaut** [comment]
Thanks I deleted the "import SDPutils" which wasn't necessary anymore and was breaking the tests.

File: test/sdp.jl [hunk 1]
```diff
@@ -2,6 +2,8 @@
 # Test SDDP functions
 ################################################################################
 using FactCheck, StochDynamicProgramming
+include("../src/SDPutils.jl")
```
Comment (tkelman) [inline test/sdp.jl]: instead of including the file here, should be able to do `using StochDynamicProgramming.SDPutils`

**Comment by trigaut** [comment]
I think we can merge but should we do it directly on master? Or on nightly?

**Comment by tkelman** [comment]
Oh, github does let you change the base branch for a pull request now. Click edit at the top if you'd like to do that.

**Comment by trigaut** [comment]
All right thanks for your advice!

**Pull request merged**